lib/webpacker/package_json.rb

```diff
@@ -0,0 +1,58 @@
+# Loads the package.json file from app root to read the webpacker configuration
+
+class Webpacker::PackageJson
```

Comment:
This whole thing is too cumbersome. I can see how it's based off configuration, but it feels very boilerplatey.